### DISCUSSION TITLE
Essentials protect not working.

### DISCUSSION THREAD

**Discussion by nafeasonto** [EssentialsX/Essentials#5293] [Q&A] [answered]
I double and triple checked the plugin is in the plugins directory. All the other plugins are working fine except for PROTECT. the protect command doesn't work, neither does the region command. It's like it's not even there, and there are NO ERRORS in the startup log for the server, no any errors in the game. What is happening? I have the latest stable build as of today on the website of Essentials and it's various plugins.

**Comment by mdcfe** [comment, marked as answer]
EssentialsX Protect lets you disable certain features across your entire server through the main EssentialsX config. You can see a list of options under the `protect:` section in `plugins/Essentials/config.yml`.

It doesn't have any region-based protection features - for that, you could consider looking into WorldGuard.
